packages/aws-cdk-lib/rosetta/aws_cloudwatch/default.ts-fixture

```diff
@@ -2,6 +2,7 @@
 import { Construct } from 'constructs';
 import { Stack, Duration } from 'aws-cdk-lib';
 import * as cloudwatch from 'aws-cdk-lib/aws-cloudwatch';
+import * as elbv2 from 'aws-cdk-lib/aws-elasticloadbalancingv2';
```

Comment:
i think this is unnecessary?

Reply:
Thank you @kaizencc !

I think it is necessary here, could you check it?

https://github.com/aws/aws-cdk/pull/30986/files#r1868706284

Reply:
https://github.com/aws/aws-cdk/pull/30986#discussion_r1868706284